biowdl_lint.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Validate every WDL file in the repository with womtool validate.
+set -e
+for WDL_FILE in $(git ls-files *.wdl)
+  do
+    echo $WDL_FILE
+    womtool validate $WDL_FILE
```

Comment:
We could consider adding `miniwdl check`?

Reply:
Yes. And some WDL-AID linting as well.